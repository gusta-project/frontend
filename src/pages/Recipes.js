import React from 'react';
import { Container, Row, Col } from 'react-bootstrap';

export default function Recipes() {
  return (
    <Container>
      <Row className="text-center">
        <Col>
          <h1>Recipes search page</h1>
        </Col>
      </Row>
    </Container>
  );
}
