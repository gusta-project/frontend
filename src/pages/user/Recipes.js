import { Helmet } from 'react-helmet';
import { Link } from 'react-router-dom';
import React, { Component } from 'react';
import { Container, Row, Col, Table } from 'react-bootstrap';

export default class UserRecipes extends Component {
  render() {
    const listItems = [];

    for (let i = 0; i < 20; i++) {
      listItems.push(
        <tr key={i}>
          <td>
            <Link to="#">Recipe {i}</Link>
          </td>
        </tr>
      );
    }

    return (
      <Container className="text-center">
        <Helmet title="Recipes" />
        <Row>
          <Col>
            <h1>Your Original Recipes</h1>
            <Table striped bordered>
              <tbody>{listItems}</tbody>
            </Table>
          </Col>
          <Col>
            <h1>Your Adapted Recipes</h1>
            <Table striped bordered>
              <tbody>{listItems}</tbody>
            </Table>
          </Col>
        </Row>
      </Container>
    );
  }
}
