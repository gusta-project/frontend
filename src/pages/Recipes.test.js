import { render } from '@testing-library/react';
import React from 'react';

import Recipes from './Recipes';

describe('<Recipes />', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Recipes />);

    expect(asFragment()).toMatchSnapshot();
  });
});
