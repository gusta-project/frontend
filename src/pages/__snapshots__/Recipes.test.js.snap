// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Recipes /> renders correctly 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="text-center row"
    >
      <div
        class="col"
      >
        <h1>
          Recipes search page
        </h1>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
